app/views/admin/bulk_uploads/set_titles.html.erb

```diff
@@ -37,12 +38,15 @@
               <%= render 'admin/attachments/reference_fields', attachable: @edition, form: attachment_fields, attachment: attachment, heading_size: "m" %>
             <% end %>
 
+            <% form.hidden_field :accessible, value: "0" %>
```

Comment:
Should this be `<%=` for it to show in the markup? 

Also I think this needs to be named uniquely for each attachment, whereas here we have multiple hidden fields of the same name on the page. In fact should it have the same name as the checkbox with value 1 so that the submitted value toggles between the two?

Reply:
Good shout. I've just realised this isn't actually needed though as it's only used to create the objects not update.